.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master, develop , ft* ]
```

Comment:
On reflection I think we probably want to save minutes and not build feature branches by default? I assume GitHub workflows have similar restrictions to Azure DevOps?